src/Ui.java

```diff
@@ -0,0 +1,149 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Ui class that deals with interactions with the user.
+ */
+public class Ui {
+    public Scanner sc;
+
+    public static String horizontal = "________________________________" + "\n";
```

Comment:
This could have been named HORIZONTAL_LINE as a constant, which will give the reader a better idea on what it is.

Reply:
And you may consider to use "final" to avoid changing this constant accidentally in the future